verse-pbc/plur
# update app icon and logo

## Issues covered
[#61](https://github.com/verse-pbc/issues/issues/61)

## Description
This PR updates the app's icon and logo. I also changed iOS logo from `.jpg` to `.png`. I only added the 3 new images and used the command ` fvm flutter pub run flutter_launcher_icons:main` to auto generate all new app icons

## How to test
1. Build the app.
2. Please check that the launch icon, app icon and logo on the login screen is the new plus one.

## Screenshots/Video

| Before | After(iOS) |  After(Android) |
| --- | --- | --- |
|<img width="250" alt="image" src="https://github.com/user-attachments/assets/a6a900cc-6913-4705-a481-5deb39b412ed">|<img src="https://github.com/user-attachments/assets/0b296bd8-11f8-4775-bce5-b82bb1339f98" alt="image" width="250"/>|<img src="https://github.com/user-attachments/assets/d25cf62b-395a-4bc3-9586-8f6c03211f2f" alt="image" width="250"/>|
|<img width="250" alt="image" src="https://github.com/user-attachments/assets/f45cd607-f6ab-48c1-ac00-67d7aeb2a343">|<img src="https://github.com/user-attachments/assets/e5fdf3e4-201d-4a53-a1c7-c79877bc9b15" alt="image" width="250"/>|<img src="https://github.com/user-attachments/assets/eb4e5766-2039-4b1f-b4cd-e89d3e839d2c" alt="image" width="250"/>|
|<img width="250" alt="image" src="https://github.com/user-attachments/assets/315a8d68-5b25-4fc3-8d99-6d7c083ff096">|<img src="https://github.com/user-attachments/assets/554ab07c-d4e6-428c-8629-4803d6e2529e" alt="image" width="250"/>|<img src="https://github.com/user-attachments/assets/ffd64480-de8f-415f-9be6-2082afa44df4" alt="image" width="250"/>|
